config/v3_1_experimental/types/schema.go

```diff
@@ -205,6 +226,11 @@ type TLS struct {
 	CertificateAuthorities []CaReference `json:"certificateAuthorities,omitempty"`
 }
 
+type Tang struct {
+	Thumbprint string `json:"thumbprint,omitempty"`
```

Comment:
Question: how will this field be filled/used? What happens if the server is advertising more than one key?

Reply:
It's a required field. It's passed in the CFG to `clevis` (e.g. `clevis luks bind -f -k $keyFile -d $device sss '{"pins": {"tang": {"url": "http://foo", "thp": "$thp"}, "t": 1}')

As far as if the server is advertising more than one key I would assume the user would pick one of the public keys and provide it.

Reply:
Thanks, this seems okay. I am not very familiar with ignition yet, but if the thp is being provided, it should be fine.